pyproject.toml

```diff
@@ -1,5 +1,67 @@
 [build-system]
-requires = ["setuptools", "wheel", "cython"]
+requires = ["setuptools"]
+
+[project]
+name="ecoscope"
+version = "1.7.3"
+description="Standard Analytical Reporting Framework for Conservation"
+readme = "README.rst"
+authors = [
+  { name = "Jake Wall", email = "walljcg@gmail.com" }
+]
+license = {file = "LICENSE"}
+classifiers=[
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.4",
+    "Programming Language :: Python :: 3.5",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+]
+dependencies = [
+    "affine",
+    "astroplan",
+    "backoff",
+    "datashader",
+    "earthengine-api",
+    "earthranger-client",
+    "folium",
+    "geopandas<=0.14.2",
+    "igraph",
+    "ipywidgets",
+    "kaleido",
+    "mapclassify",
+    "matplotlib",
```

Comment:
Eventually would be nice to split viz dependencies out as optional, but that is a separate issue 😄